examples/https-listener/https_listener.description

```diff
@@ -1,2 +1,2 @@
-// The HTTP Listener can be used to connect to, and interact with, an HTTP client.
-// Https configurations can be given to the server to expose an https connection.
+// The HTTPS Listener can be used to connect to, and interact with, an HTTPS client.
+// Https configurations can be given to the server to expose an HTTPS connection.
```

Comment:
We need to rephrase and improve this description. This sounds like a description that has not been updated since the days this was added. Sounds a bit odd.

Reply:
@chamil321 thoughts?

Reply:
Shall we make the second sentence a bit more descriptive?
Something like this "SecureSocket configuration should be given to the server to expose an HTTPS connection."